portlist/portlist_test.go

```diff
@@ -271,6 +200,11 @@ func BenchmarkGetListIncremental(b *testing.B) {
 func benchmarkGetList(b *testing.B, incremental bool) {
 	b.ReportAllocs()
 	var p Poller
+	p.init()
+	if p.initErr != nil {
+		b.Fatal(p.initErr)
```

Comment:
skip, no?

otherwise this won't pass `-benchtime=1x` on CI on all platforms.

Reply:
@bradfitz would love to learn but at this point I'm not familiar with how this repo's CI runs its benchmarks and how this change could affect it, I updated the PR nonetheless 👍🏼